drupal/web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/checkbox.twig

```diff
@@ -1,5 +1,7 @@
 <label class="checkbox {{ checkbox.class }}">
   <div class="checkbox--label">{{ checkbox.label }}</div>
-  <input type="checkbox"/>
+  {% block content %}
+    <input type="checkbox"/>
```

Comment:
Remove end tags for void elements.

Void elements (e.g. <input> tags) should not have an end tag (/>) according to 
https://www.w3.org/TR/html51/syntax.html#void-elements